tasks/eth/006-MCP-L1/postCheck.s.sol

```diff
@@ -0,0 +1,231 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
```

Comment:
We will need to add a justfile in this directory that runs this contract instead of running the base sign from json contract in the root. Perhaps we could rename this to Execute ? Thoughts on naming @maurelian @mds1

Reply:
What if we instead modify the root justfiles to look for a `PostCheck.s.sol`, and run that if present, and otherwise fallback to the original file?

Also what about renaming to something very explicit like `SignFromJsonWithAssertions.s.sol`?

Reply:
@mds1 question -- this check is only for the MCP upgrade, does it make sense to modify the root justfiles? if so, it could be as simple as adding a line that checks whether this is the MCP upgrade, and if so to look for `SignFromJsonWithAssertions`.s.sol in the task dir for MCP mainnet. the alternative, which is to just include a new justfile in the task dir, seems more elegant imo

Reply:
We will want to follow this pattern for all future upgrades and continue to improve on it over time

Reply:
Agreed with @tynes. There is a balance to strike here—some things, such as https://github.com/ethereum-optimism/superchain-ops/pull/119#discussion_r1548234133, we do not want to generalize and can hardcode. Others, such as modifying the root justfile, are worth generalizing now to avoid introducing tech debt